src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentsLoadAction.java

```diff
@@ -78,28 +78,35 @@ private void removeQuestionsAndResponsesIfNotAllowed(FeedbackSessionResultsBundl
         Iterator<FeedbackResponseAttributes> iter = bundle.responses.iterator();
         while (iter.hasNext()) {
             FeedbackResponseAttributes fdr = iter.next();
-            if (!(instructor != null &&
-                    instructor.isAllowedForPrivilege(fdr.giverSection, 
-                            fdr.feedbackSessionName, Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS)
-                    && instructor.isAllowedForPrivilege(fdr.recipientSection, 
-                            fdr.feedbackSessionName, Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS))) {
+            boolean instructorCanViewSessionInGiverSection = 
```

Comment:
can you rename these boolean variables to start with 'can'.
e.g. canInstructorViewSessionForGiverSection, canInstructorViewSessionForRecipientSection, canInstructorViewSession ...